delius-pre-prod/sub-projects/new-tech.tfvars

```diff
@@ -35,6 +35,7 @@ offenderapi_conf = {
   ecs_scaling_min_capacity = 2
   ecs_scaling_max_capacity = 10
   env_jwt_public_key       = "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0NCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBalRkdXRLS2VQUkJlQWNrZ1M2cC8NCmxWK2Q1WXFiM3ExOHo2VXdoMU1OWXhwM0hxWXBZVis5MnlGeGUyUTVSN0tWakl5S3kyZkdBanNXeStlU04yM1QNCkZURVVUeXdHdnc0SkJrRGRoaUUvKy80SjR2TWFHMzBYN29GdE9JUWIwaENFQVlWM3o4emc1WVd2OGZlRkVnWGcNCkp3SjVCc2krU0NvajNIZDlERXk1ZGZaTTljL3BoOXN5YWpLU1ZiQW8rRCtXanhQWHU4UndRSk9BU0g0NUVwclQNCjNJMy9NdllyY1ZvdkN4UVZyUWNwYzNiY0U0bHpYdFlSVy9wSWpHUVpBKy9WYjNXZXZuR1FiYndTNHp6eXkzazgNCkRCZ0pzQVJlVDI1eHp0ckptRGlZZXBHYkVKVHV0alRqR2NVOG1qUjRWZ3gwdHdqMk9DMzBES3dXbGlrMzhXTWwNCk5RSURBUUFCDQotLS0tLUVORCBQVUJMSUMgS0VZLS0tLS0="
```

Comment:
do we still need this? the public key?

Reply:
No.  But I wanted to leave it in for now just in case my changes need backing out.